docs/content/docs/tasks_usage/traffic_management/ingress_with_Azure_Application_Gateway.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
Let's move this to `traffic_management/demos`, and rename the file to `azure_application_gateway_ingress_demo.md`